### DISCUSSION TITLE
Hue Motion sensor :  "T"-Button

### DISCUSSION THREAD

**Discussion by mvogt1** [Koenkk/zigbee2mqtt#7116] [Devices]
The Hue Motion sensor has behind its rear cover a "T"-button. 
If I press this button manually and put the sensor in a group "1001" I get the following behaviour:

>Zigbee2MQTT:debug 2021-04-18 15:32:19: Received Zigbee message from '0x0017880102139444', type 'commandToggle', cluster 'genOnOff', data '{}' from endpoint 1 with groupID 1001
>Zigbee2MQTT:debug 2021-04-18 15:32:19: __No converter available__ for '9290012607' with cluster 'genOnOff' and type 'commandToggle' and data '{}'

My goal is to bind this sensor to a zigbee group, without success so far.
I'm unsure if I can ignore this "No converter available" or if this is something interesting. 
Pressing this "T"-button manually enables/disables the lamp in the zigbee group 1001 too.

Additionally I see in the log that the state changes:

> Received Zigbee message from '0x000b57fffea6ff41', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint > 1 with groupID 0
> info MQTT publish: topic 'zigbee2mqtt/1001', payload '{"state":"OFF"}'

How can I bind this sensor into a zigbee group?

**Comment by Koenkk** [comment]
If it sends message to a group you don't need to bind it anymore. Just put your device in a group with ID 1001
